src/payment-account.js

```diff
@@ -13,6 +16,68 @@ class PaymentAccount extends ExchangePaymentAccount {
       return trade;
     });
   }
+
+  sell () {
+    console.log('called sell', this);
+    const sellData = {
+      priceQuoteId: this._quote._id,
+      transferIn: {
+        medium: 'blockchain'
+      },
+      transferOut: {
+        medium: 'bank',
+        mediumReceiveAccountId: this._account.id
+      }
+    };
+    console.log('sell payload', sellData);
+    return this._api.authPOST('trades', sellData);
+  }
+
+  static add (api, obj) {
```

Comment:
I would call this `create`, not `add`. Where does `obj` come from here? If it's coming from the frontend, then you should probably add some function here that makes it easier to provide all the details needed for a bank account. It looks like you're just deep cloning the `obj` argument.